### ISSUE TITLE
Infallible.publisher should return AnyPublisher<Output, Never>

### ISSUE THREAD

**Issue by daleswift** [CombineCommunity/RxCombine#34]
RxSwift Infallible cannot fail so the publisher type should also never fail. i.e. AnyPublisher<Output, Never>

**Comment by freak4pc** [comment]
That's correct, RxCombine is pre-Infallible :) We just don't have the overloads, yet. Should be trivial to add.
If you want to work on a PR it might speed things up.

**Comment by ghost** [comment]
> RxSwift Infallible cannot fail so the publisher type should also never fail. i.e. AnyPublisher<Output, Never>
